dotcom-rendering/src/web/components/TopicFilterBank.importable.tsx

```diff
@@ -9,6 +9,7 @@ type Props = {
 	format: ArticleFormat;
 	filterKeyEvents: boolean;
 	keyEvents?: Block[];
+	urlHash: 'key-events-carousel-desktop' | 'key-events-carousel-mobile';
```

Comment:
Does it make more sense to just call this `id` everywhere?

Reply:
Yea I agree too

Reply:
cool, amended!